ReSharper.FSharp/src/FSharp/FSharp.Common/src/ProjectModel/FSharpLanguageLevelProvider.fs

```diff
@@ -97,8 +102,9 @@ type FSharpLanguageLevelProjectProperty(lifetime, locks, projectPropertiesListen
     let getFscPath (configuration: IFSharpProjectConfiguration): VirtualFileSystemPath =
         if isNull configuration then VirtualFileSystemPath.GetEmptyPathFor(InteractionContext.SolutionContext) else
 
-        let path = configuration.PropertiesCollection.GetPropertyValueSafe(FSharpProperties.DotnetFscCompilerPath)
-        VirtualFileSystemPath.TryParse(path, InteractionContext.SolutionContext)
+        match configuration.PropertiesCollection.TryGetValue(FSharpProperties.DotnetFscCompilerPath) with
+        | true, path -> path.Trim('"').ParseVirtualPathSafe(InteractionContext.SolutionContext)
```

Comment:
Isn't there a way to trim the string automatically?

Reply:
btw, in dotnet-products quotes in paths are trimmed explicitly if necessary